WebRoot/js/zimbraMail/tasks/view/ZmTaskListView.js

```diff
@@ -497,9 +487,9 @@ function(task, colIdx) {
 	htmlArr[idx++] = DwtId.getListViewItemId(DwtId.WIDGET_ITEM_FIELD, this._view, task.id, ZmItem.F_ITEM_ROW_3PANE);
 	htmlArr[idx++] = "'>";
 
-    idx = this._getAbridgedCell(htmlArr, idx, task, ZmItem.F_SUBJECT, colIdx);
+    idx = this._getAbridgedCell(htmlArr, idx, task, ZmItem.F_SUBJECT, colIdx, '', '', 'TaskSubject');
```

Comment:
I thought of doing that, but doing that will require to add more property of global object 'ZmListView'. So, instead of adding more property to global variable, went with doing this at local function level.
Let me know if you have other view